pkgs/development/python-modules/pysc2/default.nix

```diff
@@ -0,0 +1,64 @@
+{ buildPythonPackage
+, lib
+, fetchFromGitHub
+, absl-py
+, enum34
+, future
+, futures
+, mock
+, mpyq
+, numpy
+, portpicker
+, protobuf
+, pygame
+, s2clientprotocol
+, six
+, websocket_client
+, sc2-headless
+}:
+
+buildPythonPackage rec {
+  version = "1.2";
+  name = "PySC2-${version}";
+
+  src = fetchFromGitHub {
+    owner = "deepmind";
+    repo = "pysc2";
+    rev = "39f84b01d662eb58b3d95791f59208c210afd4e7";
+    sha256 = "0dfbc2krd2rys1ji75ng2nl0ki8nhnylxljcp287bfb8qyz2m25p";
+  };
+
+  patches = [
+    ./fix-setup-for-py3.patch
+    ./parameterize-runconfig-sc2path.patch
+  ];
+
+  postPatch = ''
+    substituteInPlace "./pysc2/run_configs/platforms.py" \
+      --subst-var-by 'sc2path' '${sc2-headless}'
+  '';
+
+  propagatedBuildInputs = [
+    absl-py
+    enum34
+    future
+    mock
```

Comment:
Does it need mock during runtime? Its quite unlikely. Remember, `install_requires` goes here. `tests_require` goes to `checkInputs`

Reply:
Their setup.py lists mock as an `install_requires` instead of a `tests_requires` [0]. They use it in a file that is part of their `lib` package. I suspect that it is a test that was just misplaced but it is where it is.

[0] https://github.com/deepmind/pysc2/blob/master/setup.py#L64
[1] https://github.com/deepmind/pysc2/blob/master/pysc2/lib/stopwatch_test.py